.github/renovate.json5

```diff
@@ -114,7 +114,7 @@
     },
     {
       // ensure maximum compatibility, when possible
-      "matchPaths": ["ember-toucan-core/package.json"],
+      "matchPaths": ["ember-toucan-core/package.json", "ember-toucan-form/package.json"],
```

Comment:
What is the intention of this rule?

Reply:
Hmm yeah maybe this should be changed to `"matchPaths": ["./packages/**/package.json"]` to include everything in packages?

I just (read the docs](https://docs.renovatebot.com/configuration-options/#matchpaths) and it seems like `matchPaths` works with `packageRules` to include/exclude packages from renovate checking up on.

Reply:
Example:
```
"packageRules": [
    {
      "description": "Group all dependencies from the app directory",
      "matchPaths": ["app/**"],
      "groupName": "App dependencies"
    }
  ]
```

I'm not very good at globs so take what I wrote above with a grain of salt...

Reply:
Your glob looks correct to me!